### DISCUSSION TITLE
stalled print

### DISCUSSION THREAD

**Discussion by mirlang** [mriscoc/Ender3V2S1#484] [Ideas]
I had the problem, that due to a bad cable octoprint lost the connection to the printer and the print stalled, but the nozzle stayed in top of the printed item and continued heating and melting it.

Would it be possible to detect if there are no commands received over a certain time and cancel the print + move the nozzle into a safe place?

**Comment by mriscoc** [comment]
Normally, the hotend turns off the heater when 10 minutes have passed since the last movement of the extruder.
